Codegen/test/resources/output/TypeCobol/Pointers.rdz.tcbl

```diff
@@ -5,40 +5,74 @@
        DATA DIVISION.
 
        LOCAL-STORAGE SECTION.
-      *01 pointer0 pointer.
-       01  pointer0 POINTER.
-       01 redefines pointer0.
-           03 pointer0efd9c003 pic S9(05) comp-5.
-                           
+      *01 PointerContainer typedef Strict.
+      *    02 pntr pointer.
+      *    02 pointerStat.
+      *       05 counter pic 9(5).
+      *       05 increment pic 9(2).
+      *       05 backupPointer pointer.
+
+      *01 pointer0 type PointerContainer.
+       01 pointer0.
+          02  pntr Pointer.
+          02  redefines pntr.
+              03  pntrc20a8a84 pic S9(05) comp-5.
+           02 pointerStat.
+             03 counter pic 9(5).
+             03 increment pic 9(2).
+            03  backupPointer Pointer.
+            03  redefines backupPointer.
+                04  backupPointera4fc7f95 pic S9(05) comp-5.
+                                         
       *01 pointer1 pointer.
```

Comment:
Why original line `01 pointer1 pointer` is commented/duplicated ?

Reply:
It is the way Razor replace action proceed. It comment original instruction and add the replacement below.
A solution can be no longer use Razor and the skeleton and handle the replacement of the pointer increment special case in the c# code.

Reply:
Ok, in this case, it's fine.